Servers/WindowServer/WSWindowManager.cpp

```diff
@@ -905,11 +905,14 @@ void WSWindowManager::process_mouse_event(WSMouseEvent& event, WSWindow*& hovere
             return IterationDecision::Continue;
         });
     } else {
+        bool mouse_event_in_window = false;
```

Comment:
Hmm, isn't this the same as the `hovered_window` variable?

Reply:
Oh whoops! It is too! Thanks :)